.github/workflows/continuous-integration.yml

```diff
@@ -173,6 +178,11 @@ jobs:
         run: ./scripts/ci_steps/install_native_linux_dependencies.sh
       - name: Tools Versions
         run: ./scripts/ci_steps/log_tool_versions.sh
+      - name: Configure AWS Credentials for Integration Tests
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          role-to-assume: ${{ secrets.INTEGRATION_TEST_ROLE_ARN }}
+          aws-region: us-west-2
```

Comment:
Do we need to configure AWS credentials in CI?

Reply:
Yeah CI needs credentials though they can be fake, this was just the easiest way I got it to work.

Reply:
As mentioned elsewhere, let's change this back to fake credentials.  I don't want to pull real credentials into a job that doesn't need them